Source/WebCore/Modules/indexeddb/server/MemoryIDBBackingStore.cpp

```diff
@@ -511,7 +510,7 @@ IDBError MemoryIDBBackingStore::openCursor(const IDBResourceIdentifier& transact
         if (!objectStore)
             return IDBError { ExceptionCode::UnknownError, "No backing store object store found"_s };
 
-        MemoryCursor* cursor = objectStore->maybeOpenCursor(info);
+        RefPtr<MemoryCursor> cursor = objectStore->maybeOpenCursor(info, *transaction);
```

Comment:
Perhaps change `RefPtr<MemoryCursor>` to `RefPtr`?

Reply:
`maybeOpenCursor` returns `MemoryObjectStoreCursor`, but `currentData` is public function in base class `MemoryCursor`, so I need to do cast here. Same for `maybeOpenCursor` below.

Thanks for the comment!